### PR TITLE
Bump kubevirt package version to v0.20.4

### PR DESCRIPTION
**Pull request by SchSeba**
This PR bump the kubevirt pakcage we use to v0.20.4

I also update the kubevirt and cluster-network-addons-operator deployments to latest one for the test clusters environments.

/cc @alonSadan

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -36,7 +37,14 @@ version="v1.4.7"
...
+  version = "=v0.20.4"
+
 [[constraint]]
-  name= "kubevirt.io/kubevirt"
-  source = "https://github.com/kubevirt/kubevirt.git"
-  version = "v0.17.0"
+  name = "kubevirt.io/client-go"
+  version = "=v0.20.4"
+
+[[override]]
```
Comment (phoracek) [inline Gopkg.toml]: why do we need CDI?

File: pkg/webhook/virtualmachine/virtualmachine.go [hunk 2]
```diff
@@ -110,7 +110,21 @@ func (a *virtualMachineAnnotator) mutateCreateVirtualMachinesFn(ctx context.Cont
 		"virtualMachineName", virtualMachine.Name,
 		"virtualMachineNamespace", virtualMachine.Namespace)
 
-	return a.poolManager.AllocateVirtualMachineMac(virtualMachine)
+	previousVirtualMachine := &kubevirt.VirtualMachine{}
```
Comment (phoracek) [inline pkg/webhook/virtualmachine/virtualmachine.go]: existingVirtualMachine maybe?

File: pkg/webhook/virtualmachine/virtualmachine.go [hunk 3]
```diff
@@ -110,7 +110,21 @@ func (a *virtualMachineAnnotator) mutateCreateVirtualMachinesFn(ctx context.Cont
...
+	err := a.client.Get(context.TODO(), client.ObjectKey{Namespace: virtualMachine.Namespace, Name: virtualMachine.Name}, previousVirtualMachine)
+	if err != nil {
+		// Check if the object already exist.
```
Comment (phoracek) [inline pkg/webhook/virtualmachine/virtualmachine.go]: It does the opposite. Maybe describe what are you trying to do there. "If the VM does not exist yet, allocate a new MAC address"

File: Gopkg.toml (hunk 1 above)
Comment (SchSeba) [reply Gopkg.toml]: This is an hard request by the kubevirt project and the version should be override (it takes the master) this is way is override and not in the request list of the project

**Pull request merged**